packages/kubernetes-1.27/kubernetes-1.27.spec

```diff
@@ -0,0 +1,192 @@
+
+# Build kubelet with the target toolchain.
+export KUBE_BUILD_PLATFORMS="linux/%{_cross_go_arch}"
+export %{kube_cc}
+export GOFLAGS='-tags=dockerless'
+export GOLDFLAGS="-buildmode=pie -linkmode=external -compressdwarf=false"
+make WHAT="cmd/kubelet"
+
+cd "${BUILD_TOP}/%{cpgorepo}-%{cpgover}"
+%set_cross_go_flags
+
+go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o=ecr-credential-provider cmd/ecr-credential-provider/*.go
+
+%install
+cd ../%{gorepo}-%{gover}
+output="./_output/local/bin/linux/%{_cross_go_arch}"
+install -d %{buildroot}%{_cross_bindir}
+install -p -m 0755 ${output}/kubelet %{buildroot}%{_cross_bindir}
+
+install -d %{buildroot}%{_cross_unitdir}
+install -p -m 0644 %{S:1} %{S:10} %{S:13} %{buildroot}%{_cross_unitdir}
+
+install -d %{buildroot}%{_cross_unitdir}/kubelet.service.d
+install -p -m 0644 %{S:20} %{S:21} %{S:22} %{S:23} %{buildroot}%{_cross_unitdir}/kubelet.service.d
+
+mkdir -p %{buildroot}%{_cross_templatedir}
+install -m 0644 %{S:2} %{buildroot}%{_cross_templatedir}/kubelet-env
+install -m 0644 %{S:3} %{buildroot}%{_cross_templatedir}/kubelet-config
+install -m 0644 %{S:4} %{buildroot}%{_cross_templatedir}/kubelet-kubeconfig
+install -m 0644 %{S:5} %{buildroot}%{_cross_templatedir}/kubernetes-ca-crt
+install -m 0644 %{S:6} %{buildroot}%{_cross_templatedir}/kubelet-exec-start-conf
+install -m 0644 %{S:7} %{buildroot}%{_cross_templatedir}/kubelet-bootstrap-kubeconfig
+install -m 0644 %{S:11} %{buildroot}%{_cross_templatedir}/kubelet-server-crt
+install -m 0644 %{S:12} %{buildroot}%{_cross_templatedir}/kubelet-server-key
+install -m 0644 %{S:14} %{buildroot}%{_cross_templatedir}/credential-provider-config-yaml
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -p -m 0644 %{S:8} %{buildroot}%{_cross_tmpfilesdir}/kubernetes.conf
+
+install -d %{buildroot}%{_cross_sysctldir}
+install -p -m 0644 %{S:9} %{buildroot}%{_cross_sysctldir}/90-kubelet.conf
+
+install -d %{buildroot}%{_cross_libexecdir}/kubernetes
+ln -rs \
+  %{buildroot}%{_sharedstatedir}/kubelet/plugins \
+  %{buildroot}%{_cross_libexecdir}/kubernetes/kubelet-plugins
+
+# Collect License files into one place
+cp LICENSE ../%{cpgorepo}-%{cpgover}/LICENSE.%{gorepo}
+cp third_party/forked/gonum/graph/LICENSE ../%{cpgorepo}-%{cpgover}/LICENSE.gonum.graph
+cp third_party/forked/shell2junit/LICENSE ../%{cpgorepo}-%{cpgover}/LICENSE.shell2junit
+cp third_party/forked/golang/LICENSE ../%{cpgorepo}-%{cpgover}/LICENSE.golang
+cp third_party/forked/golang/PATENTS ../%{cpgorepo}-%{cpgover}/PATENTS.golang
+
+cd ../%{cpgorepo}-%{cpgover}
+install -d %{buildroot}%{_cross_libexecdir}/kubernetes/kubelet/plugins
+install -p -m 0755 ecr-credential-provider %{buildroot}%{_cross_libexecdir}/kubernetes/kubelet/plugins/ecr-credential-provider
+mv LICENSE LICENSE.%{cpgorepo}
+
+cd ../%{gorepo}-%{gover}
+%cross_scan_attribution --clarify %{S:1000} go-vendor vendor
+
+cd ../%{cpgorepo}-%{cpgover}
+%cross_scan_attribution --clarify %{S:1000} go-vendor vendor
```

Comment:
`ecs-agent` does this differently so that the `bottlerocket-license-scan` tool only runs once - it would be good to confirm that we end up with all the expected attribution info with this approach.